resources/x509_cert_gen/gen_keystore.sh

```diff
@@ -65,20 +65,21 @@ do
 	${SSH} root@${ip} "mkdir ${INBOX}" 2>/dev/null || true
 	${SCP} chain.pem root@${ip}:${INBOX}
 	${SCP} pkey.key root@${ip}:${INBOX}
-	${SSH} root@${ip} "chmod a+x ${INBOX}${CHAIN}"
-	${SSH} root@${ip} "chmod a+x ${INBOX}${NODE}.key"
+	${SSH} root@${ip} "chmod o+rx ${INBOX}${CHAIN}"
+	${SSH} root@${ip} "chmod o+rx ${INBOX}${NODE}.key"
 
 	# Upload ROOT CA and activate it
 	curl -s -o /dev/null --data-binary "@./${ROOT_CA}.pem" \
     	http://${ADMINCRED}@${ip}:8091/controller/uploadClusterCA
 	curl -sX POST http://${ADMINCRED}@${ip}:8091/node/controller/reloadCertificate
+
 	# Enable client cert
 	if ${USE_JSON} ;then
-        	POST_DATA='{"state": "enable","prefixes": [{"path": "subject.cn","prefix": "","delimiter": ""}]}'
-        	curl -s -H "Content-Type: application/json" -X POST -d "${POST_DATA}" http://${ADMINCRED}@${ip}:8091/settings/clientCertAuth
+	    POST_DATA='{"state": "enable","prefixes": [{"path": "subject.cn","prefix": "","delimiter": ""}]}'
```

Comment:
I see 78 and 79 line are not starting at same position

Reply:
done